create-entry.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
+
+JEKYLL_ASSETS_DIR=$HOME'/Documents/portfolio/assets/'
+JEKYLL_POSTS_DIR=$HOME'/Documents/portfolio/_posts/'
+
+FOLDER_NAME='';
+TITLE='';
+DETAIL='';
+DESCRIPTION='';
+VID=false;
+IMG=false;
+
+while [ -n "$1" ]; do 
```

Comment:
Take a look at getopt -- it makes this a bit nicer: https://man.openbsd.org/getopt.1